### ISSUE TITLE
need help..

### ISSUE THREAD

**Issue by suvarna66** [android-pay/s2ap-quickstart-csharp#7]
Hi All,

 I have created sample using provided source code and  have modified the required details in Web.config for the same but  on clicking the button(save to android pay), it asks for google login, but nothing happens after clicking it, need your help for that.

![helloobjects](https://cloud.githubusercontent.com/assets/16591858/12289890/503a0700-ba04-11e5-8aad-f983358ae07a.PNG)

**Comment by samtstern** [comment]
Are there any errors appearing in the Chrome Javascript console?

**Comment by suvarna66** [comment]
ohh yaa i am getting this error in console
![helloobjects2](https://cloud.githubusercontent.com/assets/16591858/12301835/262983a2-ba48-11e5-93af-dca4f7c609fa.PNG)

**Comment by samtstern** [comment]
Ok and this is happening when you click "Insert Offer Class" and then "Save Offer"?

And you created a Web Client ID that points to localhost:59113?

**Comment by suvarna66** [comment]
yah i am getting this error on clicking on insert offer class and then on save offer...
1) I created the client id but i am not understanding, how to access this client id in code.. i got this as a client id - 234687455239-v6ditam6bhkimbre0mtfip1ms709b9el.apps.googleusercontent.com 
 service account id - shireappserviceacc@shirewalletapp.iam.gserviceaccount.com

**Comment by samtstern** [comment]
You don't need to access the Client ID in code, it just has to exist and match the project/host details.

In the Chrome inspector 'Network' tab, is there any message that comes along with the 400 or 404 errors?

**Comment by suvarna66** [comment]
1) In the Network tab  i am getting 400 and 404 errors only
![helloobject3](https://cloud.githubusercontent.com/assets/16591858/12422143/6d39b670-beeb-11e5-9acc-e79b532408ab.PNG)


2) I am getting the below error when clicking on "Insert offer"(buttton) while debugging
Error- Error:"invalid_grant", Description:"", Uri:""

**Comment by suvarna66** [comment]
hi all,
please help me in this..

**Comment by samtstern** [comment]
@suvarna66 I will see if I can get someone else to help in debugging/

**Comment by suvarna66** [comment]
okay.. Thanks samtstern

**Comment by samtstern** [comment]
Ok a few questions from the engineers:
"""
Do you know this partner's issuer ID?  I'm not sure if this developer has access to the Wallet Objects/Save to Android Pay API.

If they do have access, we need to ensure they have successfully created a class.

If they do have a class, I would guess this error is because the class is still in the draft state. This means only the developer account can save objects. 

If they change the class to "under review" which will get auto approved, then any account can save.
"""

**Comment by suvarna66** [comment]
Yes i got the issuer ID and i have access to the wallet objects/Save to Android Pay API, but i am not understanding how i will able to change the class to "under review"?
Or shall i share my code with you?

**Comment by samtstern** [comment]
Could you please email s2ap-support@google.com?  We need to get some information from you that should not be shared publicly on Github.

**Comment by suvarna66** [comment]
okay will gonna email on that.. Thank you..
